nan_persistent_12_inl.h

```diff
@@ -62,11 +63,12 @@ class Global : public v8::Global<T> {
  public:
   inline Global() : v8::Global<T>() {}
 
-  template<typename S> inline Global(v8::Local<S> that) :
+  template<typename S> inline explicit Global(v8::Local<S> that) :
     v8::Global<T>(v8::Isolate::GetCurrent(), that) {}
 
   template<typename S>
-  inline Global(const v8::PersistentBase<S> &that) :
+  inline
+  Global(const v8::PersistentBase<S> &that) :  // NOLINT(runtime/explicit)
```

Comment:
While this could be made explicit, it is not strictly necessary, as `Globals` are reset when they go out of scope. Maybe we should prefer explicit.